src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -149,4 +185,70 @@ private ConsortiaConfiguration createConsortiaConfigurationBody(String tenantId)
     configuration.setCentralTenantId(tenantId);
     return configuration;
   }
+
+  private void createShadowAdminUserWithPermissions(User user) {
+    User userOptional = userTenantService.getUser(UUID.fromString(user.getId()));
+    if (Objects.isNull(userOptional.getId())) {
+      userOptional = createUser(user);
+    }
+    Optional<PermissionUser> permissionUserOptional = permissionsClient.get("userId==" + userOptional.getId())
```

Comment:
I believe we already have exactly the same code to create or update permissions in UserTenantServiceImpl, I believe we need to inproduce PermissionsService to avoid code duplication

Reply:
fixed